### PR TITLE
styling the project

### PR DESCRIPTION
**Pull request by HanaSabih**
add style for css

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -13,7 +13,6 @@ header {
   background-color: #4caf50;
   color: white;
 }
-
```
Comment (HanaSabih) [inline style.css:16]: I think it would be better to empty the line before at-rule

File: style.css [hunk 2]
```diff
@@ -121,7 +120,7 @@ h3 {
   position: absolute;
   background-color: #f1f1f1;
   min-width: 160px;
-  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+  box-shadow: 0px 8px 16px 0 rgba(0, 0, 0, 0.2);
```
Comment (HanaSabih) [inline style.css:123]: I think there is an Unexpected unit with 0 value

**Review by HanaSabih — COMMENTED** [review]
STATUS: CHANGES REQUIRED ♻️ 🚧
While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️
You are almost there, keep up the good work! 🚀
Required Changes ♻️
why don't you add a description in the readme file
and run stylelint  to fix CSS errors.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

File: style.css (hunk 1 above)
Comment (christianbiring1) [reply style.css:16]: - [ ] Kindly make sure you always explain why a change was suggested. And also you may consider providing some online resources for further documentation :smiley:

File: style.css (hunk 2 above)
Comment (christianbiring1) [reply style.css:123]: - [ ] Kindly make sure you always explain why a change was suggested. And also you may consider providing some online resources for further documentation :smiley:

**Review by christianbiring1 — COMMENTED** [review]
Hi @HanaSabih , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] Use of GitHub inline comment to give review :+1: 
- [ ] THe comments are given in a polite way as suggestions and not as orders :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**